src/other/dz60/dz60.json

```diff
@@ -0,0 +1,645 @@
+{
+  "name": "DZ60",
+  "vendorId": "0x445A",
+  "productId": "0x2260",
+  "lighting": "qmk_backlight",
+  "matrix": {
+    "rows": 5,
+    "cols": 15
+  },
+  "layouts": {
+    "presets": {
+      "Default": [0,0,0,0],
+      "ISO": [0,1,3,0],
+      "HHKB": [0,0,0,2],
+      "With arrows": [0,0,3,3]
+    },
```

Comment:
presets aren't implemented yet

Could delete these or fix HHKB by adding 2 more bottom rows with HHKB and WKL blockers... why not? ;-)

Reply:
Is it safe to leave presets as is, or it is expected that syntax will change and it is better to delete them? 

I honestly didn't want to add hhkb/wkl variants because of how big lists already are, and IMO 7U is already perfectly usable for those cases. Now I think if we go with "layout names", having HHKB and WKL makes total sense....